janusgraph-test/src/test/java/org/janusgraph/graphdb/GraphDatabaseConfigurationInstanceIdTest.java

```diff
@@ -51,35 +45,34 @@ public void graphShouldOpenWithSameInstanceId() {
         final StandardJanusGraph graph1 = new StandardJanusGraph(new GraphDatabaseConfigurationBuilder().build(new CommonsConfiguration(config)));
 
         assertEquals(graph1.openManagement().getOpenInstances().size(), 1);
-        assertEquals(graph1.openManagement().getOpenInstances().toArray()[0], "not-unique");
+        assertEquals(graph1.openManagement().getOpenInstances().toArray()[0], "not-unique(current)");
 
         final StandardJanusGraph graph2 = new StandardJanusGraph(new GraphDatabaseConfigurationBuilder().build(new CommonsConfiguration(config)));
 
         assertEquals(graph1.openManagement().getOpenInstances().size(), 1);
-        assertEquals(graph1.openManagement().getOpenInstances().toArray()[0], "not-unique");
+        assertEquals(graph1.openManagement().getOpenInstances().toArray()[0], "not-unique(current)");
         assertEquals(graph2.openManagement().getOpenInstances().size(), 1);
-        assertEquals(graph2.openManagement().getOpenInstances().toArray()[0], "not-unique");
+        assertEquals(graph2.openManagement().getOpenInstances().toArray()[0], "not-unique(current)");
         graph1.close();
         graph2.close();
     }
 
     @Test
+    @Disabled("This should be run against a backend which support multiple backends.")
```

Comment:
Why you disabled this particular test?

Reply:
Please, use another PR to disable the test if you have an issue with it. Don't disable it in this PR because it isn't relevant to the current PR (as I understand). Currently I see that the test is enabled and works in `master`. If you must certainly disable this test when moving from `JUnit 4` to `JUnit 5` then explain why you must disable it for `JUnit 5`.